Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -243,6 +243,8 @@ def getResultSummary(self):
 
     @defer.inlineCallbacks
     def run(self):
+        if not self.reference:
+            self.reference = self.getProperty('gitmirror', None)
```

Comment:
Might be a good idea to modify `self.reference` only if `gitmirror` property is defined.

Reply:
By default it is initialized to `None` in the `__init__()` of the super class of `CheckOutSource()` (aka `buildbot.steps.source.git.Git`) so it is the same in practical terms, but I can modify the code to make clearer that only in the case of `gitmirror` property defined then self.reference will be modified.